src/processors/controllers/processorController.js

```diff
@@ -9,4 +9,17 @@ module.exports.createProcessor = function (req, res, next) {
         .catch(function (err) {
             return next(err);
         });
-};
\ No newline at end of file
+};
+
+module.exports.getAllProcessors = async function (req, res, next) {
+    let { query: { from = 0, limit = 100 } } = req;
+    let processors;
+    try {
+        from = parseInt(from);
+        limit = parseInt(limit);
+        processors = await processorManager.getAllProcessors(from, limit);
+    } catch (err) {
+        return next(err);
+    }
+    return res.status(200).json(processors);
```

Comment:
move this to line 21 after line `processors = await processorManager.getAllProcessors(from, limit);` to be aligned with how the other controllers are written